src/amo/components/Search/index.js

```diff
@@ -120,10 +124,19 @@ export class SearchBase extends React.Component {
       <SearchSort filters={filters} pathname={pathname} />
     ) : null;
 
+    if (!queryParams.q) {
+      return (
+        <div className="Search">
+          <Card className="SearchContextCard">
+            <h1 className="SearchContextCard-header">{i18n.gettext('Enter a search term and try again.')}</h1>
+          </Card>
+        </div>
+      );
+    }
+
     return (
       <div className="Search">
         {errorHandler.renderErrorIfPresent()}
-
```

Comment:
Why was this removed? The whitespace here was used to divide up different sections for readability.